_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -156,120 +136,120 @@ The `input` element does not have a semantic role that is a widget role, but sti
 
 #### Failed Example 1
 
-Unknown autocomplete term.
+This `autocomplete` [attribute value][] does not have a [correct autocomplete field][].
```

Comment:
I think sometimes the previous short phrases were a little clearer about certain things. In this case because 'not correct' doesn't convey quite the same thing as 'unknown'.

```suggestion
This `autocomplete` [attribute value][] has an unknown term that is not a [correct autocomplete field][].
```